include/libopencm3/stm32/f4/rcc.h

```diff
@@ -790,6 +795,7 @@ struct rcc_clock_scale {
 	uint32_t apb2_frequency;
 };
 
+extern const struct rcc_clock_scale rcc_hsi_configs[RCC_CLOCK_3V3_END];
```

Comment:
I'm not really enthused by having a bunch of separate lists, especially when they all seem to use the same end address, but yeah, it's one way of doing it.

Reply:
What would you propose? A function instead? Is there an example in other families?

Reply:
@karlp Should we mark this as resolved then? May I proceed with squashing the fix-up commits?

Reply:
nah, it's all good, I've merged and squished them locally, just doign some other tests and it will probably land within the hour :)